pyro/distributions/conditional.py

```diff
@@ -3,6 +3,7 @@
 
 from abc import ABC, abstractmethod
 
+import pyro
```

Comment:
Could you avoid the top level import and instead
```py
from .torch import TransformedDistribution
```
and use `TransformedDistribution` below? The reason is that `pyro.distributions` tries to be self-contained and avoid ever importing `pyro`.

Reply:
Sure thing! Just changed that.